### PR TITLE
Run Prettier over String/Date pages

### PR DESCRIPTION
**Pull request by Josh-Cena**

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/string/split/index.md [hunk 1]
```diff
@@ -89,27 +89,32 @@ number of elements in the array, and the individual array elements.
...
-splitString(monthString, comma)
+splitString(tempestString, space);
+splitString(tempestString);
+splitString(monthString, comma);
 ```
 
 This example produces the following output:
 
-```plain
+```
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/string/split/index.md:117]: Why this change? I don't care but sometimes people like to add `plain` in cases like this.

File: files/en-us/web/javascript/reference/global_objects/string/split/index.md [hunk 2]
```diff
@@ -131,20 +136,20 @@ the semicolon from the string. `nameList` is the array returned as a result
...
 
-console.log(nameList)
+console.log(nameList);
 ```
 
 This logs two lines; the first line logs the original string, and the second line logs
 the resulting array.
 
-```plain
+```
```
Comment (wbamberg) [inline files/en-us/web/javascript/reference/global_objects/string/split/index.md:152]: also

**Review by wbamberg — APPROVED** [review]
👍 thank you @Josh-Cena ! I had questions but they are not critical.

File: files/en-us/web/javascript/reference/global_objects/string/split/index.md (hunk 1 above)
Comment (Josh-Cena) [reply files/en-us/web/javascript/reference/global_objects/string/split/index.md:117]: Sorry @wbamberg I was a bit occupied yesterday. There's perhaps a bit more reasoning in https://github.com/mdn/content/pull/20030. TLDR: I believe `plain` is always an editorial mistake, intentional or unintentional, because it doesn't map to any computer language. It's just an invalid language with a good name that makes people think it exists.

Comment (wbamberg) [reply files/en-us/web/javascript/reference/global_objects/string/split/index.md:117]: yeah, I agree with that 👍
